packit_service/worker/handlers/distgit.py

```diff
@@ -406,6 +406,23 @@ def __init__(
         )
         self.task = task
         self.dg_branch = event.get("git_ref")
+        self._pull_request: Optional[PullRequest] = None
+
+    @property
+    def pull_request(self):
+        if not self._pull_request and self.data.event_dict["committer"] == "pagure":
+            prs = [
+                pr
+                for pr in self.project.get_pr_list(status=PRStatus.merged)
+                if pr.head_commit == self.data.commit_sha
+            ]
+            if prs:
+                self._pull_request = prs[0]
```

Comment:
should there be an explicit `else:` path here to indicate that the commit is not coming from a PR?

Reply:
update: discussed this over chat; status: resolved :)